modules/media/jsx/columnFormatter.js

```diff
@@ -18,12 +18,19 @@ function formatColumn(column, cell, rowData, rowHeaders) {
     row[header] = rowData[index];
   }, this);
 
+  // create array of classes to be added to td tag
+  var classes = [];
+  if (row['Hide File'] === '1') {
+    classes.push("bg-danger");
+  }
+  // convert array to string, with blank space separator
+  classes = classes.join(" ");
```

Comment:
@johnsaigle  I recall @ridz1208  saying he discussed this with Dave, so I'm going to move this Needs review by senior dev